### ISSUE TITLE
incorrect Swish

### ISSUE THREAD

**Issue by n01z3** [katsura-jp/efficientnet-pytorch#4]
According to [the article](https://arxiv.org/abs/1710.05941) Swish should be `input * torch.sigmoid(beta * input)`, where beta - trainable. 
Your realization has no beta param.

**Comment by katsura-jp** [comment]
thank you for teach me about it. 
After reading the paper, implement and republish it.

**Comment by vfdev-5** [comment]
According to official tf repo of efficientnet, swish is without trainable beta : 

https://github.com/tensorflow/tpu/blob/05f7b15cdf0ae36bac84beb4aef0a09983ce8f66/models/official/efficientnet/efficientnet_model.py#L44

https://github.com/tensorflow/tensorflow/blob/2d29022cfb2f3b352ec8a85df301a63d578e96dc/tensorflow/python/ops/nn_impl.py#L435

**Comment by katsura-jp** [comment]
thank you for sharing:D
I will implement it by selecting whether the parameter of swish is trainable or constant depending on the argument.

**Comment by katsura-jp** [comment]
Added trainable parameter, β to swish.
